models/appointment.js

```diff
@@ -34,17 +34,18 @@ AppointmentSchema.statics.sendNotifications = function(callback) {
 
     // Send messages to all appoinment owners via Twilio
     function sendNotifications(docs) {
```

Comment:
This method should return a promise. Otherwise if there is an error it will be unhandled.